tools/src/main/scala/org/apache/spark/sql/rapids/tool/profiling/EventsProcessor.scala

```diff
@@ -246,75 +248,86 @@ object EventsProcessor extends Logging {
       app: ApplicationInfo,
       event: SparkListenerTaskStart): Unit = {
     logDebug("Processing event: " + event.getClass)
-    app.taskStart += event
+    // currently not used
+    // app.taskStart += event
   }
 
   def doSparkListenerTaskEnd(
       app: ApplicationInfo,
       event: SparkListenerTaskEnd): Unit = {
     logDebug("Processing event: " + event.getClass)
 
-    // Parse task accumulables
-    for (res <- event.taskInfo.accumulables) {
-      try {
-        val value = res.value.getOrElse("").toString.toLong
-        val thisMetric = TaskStageAccumCase(
-          event.stageId, event.stageAttemptId, Some(event.taskInfo.taskId),
-          res.id, res.name, Some(value), res.internal)
-        app.taskStageAccum += thisMetric
-      } catch {
-        case e: ClassCastException =>
-          logWarning("ClassCastException when parsing accumulables for task "
+    if (forQualification) {
+      val stageAndAttempt = s"${event.stageId}:${event.stageAttemptId}"
+      val taskSum = app.stageTaskQualificationEnd.getOrElseUpdate(stageAndAttempt, {
+        new StageTaskQualificationSummary(event.stageId, event.stageAttemptId, 0, 0)
+      })
+      taskSum.executorRunTime += event.taskMetrics.executorRunTime
+      taskSum.executorCPUTime += NANOSECONDS.toMillis(event.taskMetrics.executorCpuTime)
+    } else {
+
+      // Parse task accumulables
+      for (res <- event.taskInfo.accumulables) {
+        try {
+          val value = res.value.getOrElse("").toString.toLong
+          val thisMetric = TaskStageAccumCase(
+            event.stageId, event.stageAttemptId, Some(event.taskInfo.taskId),
+            res.id, res.name, Some(value), res.internal)
+          app.taskStageAccum += thisMetric
+        } catch {
+          case e: ClassCastException =>
+            logWarning("ClassCastException when parsing accumulables for task "
```

Comment:
Since we are already modify this code, maybe let us combine it into a single log statement for the same issue instead of three

Reply:
same here, it's not required right now so we can do followup.  I would like to make pass through all code next release to clean some things up